src/components/visualizations/table/totals/utils.ts

```diff
@@ -18,15 +18,39 @@ export const isNativeTotal = (total: AFM.ITotalItem) => {
     return total && total.type === "nat";
 };
 
-export const getNativeTotals = (totals: AFM.ITotalItem[]): AFM.INativeTotalItem[] => {
+export const getAttributeDimension = (
+    attributeIdentifier: string,
+    resultSpec: AFM.IResultSpec,
+): AFM.IDimension => {
+    return (
+        resultSpec.dimensions.find(
+            dimension => !!dimension.itemIdentifiers.find(attribute => attribute === attributeIdentifier),
+        ) || null
```

Comment:
this returns null when dimension is not found... but then result is use in functions
`isGrandTotal`
`getAllSubtotalsOfTypeFromDimension` which are not expecting null as dimension...